src/commands/game_options/cutoff.ts

```diff
@@ -11,8 +11,11 @@ export default class CutoffCommand implements BaseCommand {
     async call({ message, parsedMessage }: CommandArgs) {
         const guildPreference = await getGuildPreference(message.guildID);
         const yearRange = parsedMessage.components;
-        const startYear = yearRange[0];
-        if (yearRange.length === 1) {
+        const startYear = yearRange[0] || null;
```

Comment:
What's the purpose of the `|| null` here?

Reply:
parsedMessage.components at 0 may be null when no arguments are passed

Reply:
Am I using it wrong?

Reply:
You already have a `yearRange.length === 0` check below, so it would never be null where it matters. Alternatively, make a this `=== 0` check a separate if statement with an early return, then evaluate `startYear` after

Reply:
My mistake, thanks for the explanation! I'll do the early return